### PR TITLE
MEPTS 1361 | TRF_IN Changes Dec. 2023 Release 

### PR DESCRIPTION
**Pull request by TinoMuianga**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public CohortDefinition getTransferredInPatients() {
     cd.addSearch("homeVisitTrfOut", EptsReportUtils.map(homeVisitTrfOut, mappingsHomeVisitTrfOut));
     cd.addSearch("clinicalVisit", EptsReportUtils.map(clinicalVisit, mappingsClinicalVisit));
 
-    cd.setCompositionString("(transferredIn OR transferredOut) AND clinicalVisit AND NOT txCurr");
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java:110]: This criteria should remain. You should check if the patients had a pick up during the period

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java [hunk 2]
```diff
@@ -201,7 +201,6 @@ public CohortDefinition getPatientsTrfOut() {
     sqlCohortDefinition.setName(
         "Patient Transferred Out With No Drug Pick After The Transferred out Date ");
     sqlCohortDefinition.addParameter(new Parameter("startDate", "Start Date", Date.class));
-    sqlCohortDefinition.addParameter(new Parameter("endDate", "End Date", Date.class));
     sqlCohortDefinition.addParameter(new Parameter("location", "Location", Location.class));
 
     Map<String, Integer> map = new HashMap<>();
```
Comment (eliazmuchave) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java]: This transferred out cohort is not aligned with the one used in Tx curr. Check the program criteria.
Check which implementation is a aligned with the spec

"Dev: confirm the code is using the same definition of transferred out from Tx Curr ( here it should be by end of previous reporting period.)"

**Review by eliazmuchave — CHANGES_REQUESTED** [review]
Left comments. Have a look

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java (hunk 1 above)
Comment (TinoMuianga) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TransferredInCohortQueries.java:110]: Fixed: Implemented the filter of drug_pickup

**Review by eliazmuchave — APPROVED** [review]
Looks good to me

**Review by eliazmuchave — APPROVED** [review]
Looks good to me

**Pull request merged**